_posts/2017-10-08-lil-yachty.md

```diff
@@ -75,7 +76,7 @@ also includes the dollar cost of his brush with the law, which although is
 insignificant compared to prison time, is not nothing.
 
 Throughout *Minnesota* Lil Yachty appeals directly to his listeners.
-In fact, he reprimands “you”, asking 
+In fact, he reprimands “you”, asking
```

Comment:
hm does your editor automatically remove trailing whitespace?

Reply:
oh yea, looks like Atom was doing that; I added the space back in